docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -4,228 +4,226 @@ Install a nested OCP cluster running on VMs within a management OCP cluster
 
 ## Prerequisites
 
-* The HyperShift CLI (`hypershift`).
+* Admin access to an OpenShift cluster (version 4.12+) specified by the `KUBECONFIG` environment variable.
+* The management OCP cluster must have wildcard dns routes enabled.
 
-    Install it using Go 1.18+:
-       ```shell
-       go get -u github.com/openshift/hypershift@latest
        ```
+       oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'
+       ```
+* The management OCP cluster must have Openshift Virtualization installed on it. [Instructions for installing Openshift Virtualization](https://docs.openshift.com/container-platform/4.12/virt/about-virt.html?extIdCarryOver=true&sc_cid=7013a0000026OSTAA2)
+* The Management OCP cluster must be configured with OVNKubernetes as the default pod network CNI.
+* The Management OCP cluster must have LoadBalancer service support. [Instructions for installing MetalLB](https://docs.openshift.com/container-platform/4.12/networking/metallb/metallb-operator-install.html)
+* The management OCP cluster must have default storage class. [Storage Configuration Documentation](https://docs.openshift.com/container-platform/4.12/post_installation_configuration/storage-configuration.html)
+    Below is an example of how to annotate a storageclass as the default storageclass.
```

Comment:
The example given is to annotate one of the storage classes managed by ODF. Perhaps we wanna emphasize `ODF storageclass` . Here is the [installation guide](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/deploying_openshift_data_foundation_using_bare_metal_infrastructure/deploy-using-local-storage-devices-bm) if you think it's necessary to include it somewhere.

Reply:
I don't think I'm ready to talk about ODF quite yet. We'll need a RWX csi driver once we support live migration, but we're about 5 months away from that being a reality.